Dockerfile

```diff
@@ -49,8 +49,5 @@ WORKDIR /app
 RUN pip install pycairo
 RUN pip install --requirement /app/requirements.txt
 
-# install aubio 0.4.9 from source for ffmpeg
-RUN git clone https://git.aubio.org/aubio/aubio aubio_src && cd aubio_src && git checkout 0.4.9 && ./setup.py clean && pip install -v .
```

Comment:
Did you test the docker file when aubio is installed from pypi?

Reply:
Not yet, as I have no ready docker installation at hand.

Reply:
But I can check tomorrow.